babel.so/src/rvs_stress.cpp

```diff
@@ -70,35 +71,37 @@ void run_stress(std::pair<int, uint16_t> device, int num_times, int ARRAY_SIZE,
 {
   std::string   msg;
   std::streamsize ss = std::cout.precision();
-
+  std::stringstream sstr;
   if (!output_as_csv)
   {
-    std::cout << "Running kernels " << num_times << " times" << std::endl;
+    msg = "Running kernels " + std::to_string(num_times) + " times, " ;
 
 
     if (sizeof(T) == sizeof(float)) 
-      std::cout << "Precision: float" << std::endl;
+      msg += "Precision: float";
     else
-      std::cout << "Precision: double" << std::endl;
+      msg += "Precision: double";
 
+    rvs::lp::Log(msg, rvs::logresults);
     if (mibibytes)
     {
       // MiB = 2^20
-      std::cout << std::setprecision(1) << std::fixed
+      sstr << std::setprecision(1) << std::fixed
                 << "Array size: " << ARRAY_SIZE*sizeof(T)*pow(2.0, -20.0) << " MiB"
-                << " (=" << ARRAY_SIZE*sizeof(T)*pow(2.0, -30.0) << " GiB)" << std::endl;
-      std::cout << "Total size: " << 3.0*ARRAY_SIZE*sizeof(T)*pow(2.0, -20.0) << " MiB"
+                << " (=" << ARRAY_SIZE*sizeof(T)*pow(2.0, -30.0) << " GiB), ";
```

Comment:
There is a "," at the end. Do we need it ?

Reply:
yes, it is a better delimiter than a newline since we will combine logs of one GPU as a virtually single log line